src/main/java/com/gw/exercises/task1/HelloName.java

```diff
@@ -0,0 +1,10 @@
+package com.gw.exercises.task1;
+
+public class HelloName {
+    public static void main (String[] args) {
+        //Write programm that returns to display word "Hello". Then from new line your name.
+        String greetMsg = "Hello";
+        String name = "Konstantine";
+        System.out.println(greetMsg + "\n" + name);
+    }
+}
```

Comment:
All code from the task 001 doesn't have to be commited in task 002. Therefore you must delete from branch 002 code which already exists in the repo and push only new code changes.